lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -158,7 +158,13 @@ function _reportback_item_resource_index($campaigns, $exclude, $status, $count,
 
   if (variable_get('rogue_collection', FALSE)) {
     $client = dosomething_rogue_client('v1');
-    $response = $client->getAllReportbacks($inputs);
+    $response = $client->getAllReportbacks([
+      'filter' => [
+        'campaign_id' => $campaigns,
+      ],
+      'as_user' => dosomething_user_get_field('field_northstar_id'),
+    ]);
+
     return $response;
```

Comment:
@DFurnes this query works however, it is still returning the first 6 posts that we already have shown and gives all the remaining posts, not just the next 6. Not sure how to write to show only next 6?